austation/code/modules/surgery/organs/tongue.dm

```diff
@@ -0,0 +1,75 @@
+/obj/item/organ/tongue
+	var/static/list/austation_languages_possible_base = typecacheof(list(
+		/datum/language/moffic,
+		/datum/language/sylvan,
+		/datum/language/shadowtongue,
+		/datum/language/terrum,
```

Comment:
not entirely sure, but is this supposed to be missing buzzwords, voltaic and calcic?

Reply:
i'm just going off what tg put in their code, they didn't have those three either

Reply:
hmm, alright I'll investigate further